package.json

```diff
@@ -36,8 +36,8 @@
     "dotenv": "5.0.1",
     "form-data": "2.2.0",
     "lodash": "4.17.10",
-    "node-fetch": "1.7.1",
-    "zapier-platform-schema": "6.1.0"
+    "node-fetch": "2.1.2",
+    "zapier-platform-schema": "7.0.0"
```

Comment:
this gets bumped automatically on release, so we shouldn't change it manually here.